lab_3_generate_by_ngrams/main.py

```diff
@@ -41,6 +43,28 @@ def _tokenize(self, text: str) -> Optional[tuple[str, ...]]:
         In case of corrupt input arguments, None is returned.
         In case any of methods used return None, None is returned.
         """
+        if not isinstance(text, str):
+            return None
+
+        text = text.lower()
+        tokens = []
+
+        for word in text.split():
+            word = ''.join(letter for letter in word if letter.isalpha())
+            if list(word):
```

Comment:
Please, describe what do you want to check in this line?

Reply:
I want to check whether every letter in a word is indeed a letter and not a number or punctuation mark.